beta/tests/tensorflow/test_weekly.py

```diff
@@ -0,0 +1,278 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import json
+import os
+import tempfile
+
+import pytest
+from pytest import approx
+
+from beta.tests.conftest import PROJECT_ROOT
+from beta.tests.tensorflow.test_sanity_sample import get_sample_fn
+
+
+EXAMPLES_DIR = PROJECT_ROOT.joinpath('examples', 'tensorflow')
+
+# sample
+# ├── dataset
+# │   ├── path
+# │   ├── batch
+# │   ├── configs
+# │   │     ├─── config_filename
+# │   │     │       ├── expected_accuracy
+# │   │     │       ├── absolute_tolerance_train
+# │   │     │       ├── absolute_tolerance_eval
+# │   │     │       ├── execution_arg
+# │   │     │       ├── weights
+GLOBAL_CONFIG = {
```

Comment:
Minor comment: what are the reasons behind splitting configs on `sota_checkpoints_eval.json` and `GLOBAL_CONFIG`? I mean, it's look like duplication, and if one would like to change target accuracy he should change it in both at `sota_checkpoints_eval.json` and `GLOBAL_CONFIG`

Reply:
Yes, right now it's duplication.
We are planning to add configs for cifar100 to `GLOBAL_CONFIG` and run it  weekly on CI.
For object detection and segmentation models, we need to specify FP32 weights, set `absolute_tolerance_train` and `absolute_tolerance_test`.